generate/tabular.go

```diff
@@ -0,0 +1,104 @@
+// Package generate is for generating random data from given structures
+package generate
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/qri-io/dataset"
+	"github.com/qri-io/dataset/dsio"
+	"github.com/qri-io/dataset/tabular"
+)
+
+// Config stores settings for the generate package.
+type Config struct {
+	random        *rand.Rand
+	maxLen        int
+	useRandomType bool
+}
+
+// DefaultConfig returns the default configuration for a Generator.
+func DefaultConfig() *Config {
+	return &Config{
+		random:        rand.New(rand.NewSource(time.Now().UnixNano())),
+		maxLen:        64,
+		useRandomType: false,
+	}
+}
+
+// AssignSeed sets a specific random seed to be used.
+func AssignSeed(cfg *Config) {
+	cfg.random = rand.New(rand.NewSource(4))
```

Comment:
Instead of `4` I'd like to have this configurable with a sane default fallback. Allows for having multiple but repeatable generated files later.

Reply:
Totally agreed. For the sake of getting this merged, I'm going to punt this until later.